.github/workflows/automaticSemVer.yml

```diff
@@ -0,0 +1,26 @@
+name: Semantic Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '14'
+
+      - name: Install semantic-release
+        run: npm install -g semantic-release@17 @semantic-release/git@10
+
+      - name: Semantic Release
+        run: npx semantic-release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Should this file be part of the commit?

Reply:
Yes, this GitHub Action is a better alternative to the previous versioning action that I have told you about previously.

This one will analyze the commits of PRs and determine the next best version to release. I forgot to add a configuration file for it as well, which I will do in this next commit.